script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -861,17 +910,18 @@ Come to think of it, I've never spent time in the courtyard like this before, co
 [sha:3e6b08507956874a781a68ebf066b1244923a81f]{
 -- Page 119, Offset 3901.
 -- あんまり中庭に出た事がなかったから。ああでも、前にもこんなコトがあったような。シエル先輩と会った時でしたっけ？」
+// Wait, is this a reference to the OG?
```

Comment:
:aokothink:

Reply:
I've noticed a couple of lines like this and it scares me. red garden is gonna ff7r this shit i can feel it. higurashi did it too